ush/setup.sh

```diff
@@ -436,7 +436,7 @@ case $MACHINE in
     QUEUE_DEFAULT=${QUEUE_DEFAULT:-"batch"}
     PARTITION_HPSS=${PARTITION_HPSS:-"service"}
     QUEUE_HPSS=${QUEUE_HPSS:-"batch"}
-    PARTITION_FCST=${PARTITION_FCST:-"kjet"}
+    PARTITION_FCST=${PARTITION_FCST:-"sjet,vjet,kjet,xjet"}
```

Comment:
This goes with the retro setup and not using reservations, correct?

Reply:
Yes, I will need to make more changes to use "jet" for real-time and the full list for retro.